src/main/resources/db/migration/V7__add_id_to_user_session.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `user_session`
+    ADD COLUMN `id` BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY;
```

Comment:
What are you doing? This has no sense

Reply:
that was the part where after removing the wrong primary key, I insert the correct primary key